### PR TITLE
 PE-436: each example should have its own README to explain and instruct usage

### PR DESCRIPTION
**Pull request by jdmedlin1**
## 📝 Description

This PR adds descriptive changes to existing readmes in effort to better inform and instruct users about the examples provided in this repository. 

The two primary changes in this PR are 1) the node-simple-server-example readme to add build and run instructions, and 2) context of other examples in the root of the examples directory.

Issue: https://brightsign.atlassian.net/browse/PE-436

## 📋 List of Changes

- Please provide a concise list of the main changes.

## 🧪 Steps to Test

1. Step 1

## ✔️ Dev Complete Checklist

- [ ] PR template filled out
- [ ] Change is tested by submitter
- [ ] PR follows all linting and coding standards
- [ ] Github Issue exists (if applicable)
- [ ] Team member has been assigned
- [ ] At least one commit message is in [Conventional Commit](https://www.conventionalcommits.org/) format

### CODE REVIEW

**Comment by mingjun7** [comment]
PR description needs some update?

**Comment by mingjun7** [comment]
I'm a bit confused what this PR is intend to do. From the title it sounds like there are some repo does not have readme, and the task is to add the readme to some example folder. The changes are modifying the readme instead. Can you describe what are the points need update in the existing readme? Does any other file need to be updated?

Also, I think we should consolidate the file name readme.md v.s. README.md across the repo.

File: examples/README.md [hunk 1]
```diff
@@ -20,11 +20,15 @@ Once you are comfortable with HTML, move on to the `starter-node-example`. This
...
 
-After mastering the basics of HTML and Node.js, you can explore the templates in the `templates` directory. These templates provide more advanced setups depending on the type of application you plan to create.
+After getting familiar the basics of HTML and Node.js, you can explore the templates in the `templates` directory. These templates provide more advanced setups depending on the type of application you plan to create.
 
 - **Location**: `templates`
 - **Instructions**: Choose a template that fits your application needs and follow the specific instructions provided in the template's README file.
 
+### 4. Other Examples
+
+Other examples in this directory intend to offer more specific references for applications. Once you get your bearing with starter examples, more tools and specific use cases will be offered. For example, applications will include tests, mocks, webpack configurations, scripts to push code to your player seemlessly, and more.
```
Comment (blshukla) [inline examples/README.md:30]:
```suggestion
Other examples in this directory intend to offer more specific references for applications. Once you get your bearing with starter examples, more tools and specific use cases will be offered. For example, applications will include tests, mocks, webpack configurations, scripts to push code to your player seamlessly, and more. 
```

File: examples/node-simple-server-example/readme.md [hunk 2]
```diff
@@ -1,12 +1,45 @@
-# Hosting a Node.js server on your player
+# Hosting a Node.js Server Example
 
-This example shows you how to host a node.js server on your player. In this example, the server serves the device info object from the `@brightsign/deviceinfo` API when a request is made to the player on port `13131`.
+## Overview
 
-## Deployment
+This example is designed to help you get started with developing on a BrightSign player using Node.js. It demonstrates how to run a simple Node.js server directly on the player. Additionally, it introduces tools and configurations that go beyond the basics covered in starter examples.
```
Comment (blshukla) [inline examples/node-simple-server-example/readme.md:5]: That first sentence sounds strange. - "developing" what?

**Review by blshukla — APPROVED** [review]
Added some nitpicks, looks good otherwise.

**Review by blshukla — APPROVED** [review]

**Pull request merged**